catalog/mvcc/views.go

```diff
@@ -14,11 +14,17 @@ const (
 )
 
 func sqEntriesV(requestedCommit CommitID) sq.SelectBuilder {
+	var actualCommit CommitID
+	if requestedCommit == UncommittedID || requestedCommit == CommittedID {
+		actualCommit = MaxCommitID
+	} else {
+		actualCommit = requestedCommit
+	}
 	entriesQ := sq.Select("*",
 		fmt.Sprintf("min_commit != %d AS is_committed", MinCommitUncommittedIndicator),
 		"max_commit = 0 AS is_tombstone",
 		"ctid AS entry_ctid\n",
-		fmt.Sprintf("max_commit < %d AS is_deleted", MaxCommitID)).
+		fmt.Sprintf("max_commit < %d AS is_deleted", actualCommit)).
```

Comment:
Use the Column syntax - ex: .Column("min_commit != ? AS is_committed", MinCommitUncommittedIndicator)